package/src/cli/commands/generate/generateDistFiles.test.ts

```diff
@@ -183,7 +181,12 @@ export const clientMessages = {
 	"en": {
 		"namespace.message": "Namespaced message"
 	}
-};
+} as const;
+
+export const messages = {
+	"en": { ...serverOnlyMessages["en"], ...clientMessages["en"] },
+	"fi": { ...serverOnlyMessages["fi"], ...clientMessages["fi"] }
```

Comment:
Order of these needs to be swapped for test to pass.

```suggestion
	"fi": { ...serverOnlyMessages["en"], ...clientMessages["fi"] },
	"en": { ...serverOnlyMessages["fi"], ...clientMessages["en"] }
```